docs/guide/tutorial-8-using-ros.md

```diff
@@ -14,23 +14,35 @@ To install the latest version of ROS on Ubuntu use the following commands:
 sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
 sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
 sudo apt-get update
-sudo apt-get install ros-melodic-desktop-full
+sudo apt-get install ros-melodic-desktop-full # takes time, get a coffee :)
 sudo rosdep init
 rosdep update
+sudo apt-get install ros-melodic-webots-ros
 ```
 
 For more information or to install it on another platform please read [http://wiki.ros.org/ROS/Installation](http://wiki.ros.org/ROS/Installation).
 Unless you need older version for some other application, you should choose the latest distribution (Melodic Morenia or Kinetic Kame).
 
-If you never used the ROS framework before, it is strongly recommended to follow some tutorials from: [http://wiki.ros.org/ROS/Tutorials](http://wiki.ros.org/ROS/Tutorials).
+The last line is to install the [webots\_ros](http://wiki.ros.org/webots\_ros) package.
+
+If you never used the ROS framework before, it is strongly recommended to follow some tutorials (_Beginner Level_) from: [http://wiki.ros.org/ROS/Tutorials](http://wiki.ros.org/ROS/Tutorials).
 These tutorials will also help you set up your ROS environment and initialize your catkin workspace.
+The minimum requirement is to follow these instructions (taken from the [ROS tutorial 1.1.1](http://wiki.ros.org/ROS/Tutorials/InstallingandConfiguringROSEnvironment)).
+1. Open a terminal
+2. Check if the environment is source : `printenv | grep ROS`
+3. If not, source it : `source opt/ros/melodic/setup.bash`. Check it again to be sure.
+4. Create a `catkin_ws` workspace for ROS :
+  1. Create some folders : `mkdir -p ~/catkin_ws/src`
+  2. Go back in the main folder : `cd ~/catkin_ws/`
+  3. Generate files by compilation : `catkin_make`
+  4. Source again the new files : `source devel/setup.bash`
+5. To make sure your workspace is properly overlayed by the setup script, make sure `ROS_PACKAGE_PATH` environment variable includes the directory you are in. The commande `echo $ROS_PACKAGE_PATH` should return `/home/youruser/catkin_ws/src:/opt/ros/melodic/share`.
+6. Last step is to set the [`WEBOTS_HOME`](https://cyberbotics.com/doc/guide/compiling-controllers-in-a-terminal) path : `export WEBOTS_HOME=/usr/local/webots`. If you installed Webots in a different place, adapt the path. This line can be add to your `.bashrc` file.
```

Comment:
Last step &rarr; The last step
"path :" &rarr; "environment variable:"
path &rarr; variable
add &rarr; added